documentation/doc-Migration_Toolkit_for_Virtualization/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -106,27 +106,27 @@ spec:
   map:
     networks:
       - source:
-          id: <network_ref> <3>
+          id: <source_network> <3>
         destination:
           type: pod
           name: pod
           namespace: <target_namespace> <4>
     datastores:
       - source: <5>
-          id: <datastore_ref> <6>
+          id: <source_data_store> <6>
         destination:
           storageClass: standard
   vms:
-    - id: <vm_ref_or_name> <7>
+    - id: <source_vm> <7>
 EOF
 ----
 <1> Specify the target cluster.
 <2> Specify the target namespace.
-<3> Specify the managed object reference of one or more VMware networks.
+<3> Specify the managed object reference or name of the source network.
```

Comment:
1. There can be more than one Network. (For example if a VM has 2 nic on 2 different networks, both needs to be mapped).
2. It is not clear from this line that to use name and not id, need to actually write it like this:
- source:
       name: <Network name>